feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt

```diff
@@ -37,25 +37,36 @@ import kotlinx.coroutines.launch
 @Suppress("LongMethod")
 internal fun HomeScreenUI(
     state: HomeState,
+    timerState: TimerState,
     bottomSheetState: AnimealBottomSheetState,
     onScreenEvent: (HomeScreenEvent) -> Unit,
-    onTimerExpire: () -> Unit
+    onTimerEvent: (TimerEvent) -> Unit
 ) {
     val (contentAlpha: Float, buttonAlpha: Float) = bottomSheetState.contentAlphaButtonAlpha()
     val scope = rememberCoroutineScope()
 
-    if (state.feedingRouteState is FeedingRouteState.TimerExpired) {
-        FeedingExpiredDialog(
-            onConfirm = {
-                onScreenEvent(HomeScreenEvent.RouteEvent.FeedingRouteCancellationRequest)
-            }
-        )
+    val hideBottomSheet: () -> Unit = {
+        scope.launch {
+            bottomSheetState.hide()
+        }
     }
 
-    scope.launch {
```

Comment:
I'm uncertain that this condition was needed, as it completely blocks viewing bottom sheet during showing route to feeding points, according to my ticket to Figma designs accessing bottom sheet should be possible. Please, correct me if I'm wrong and it should remain